packages/react-app-revamp/hooks/useContest/index.ts

```diff
@@ -295,6 +316,40 @@ export function useContest() {
     }
   }
 
+  async function checkCurrentUserAmountOfProposalTokens() {
+    const abi = await getContestContractVersion(address);
```

Comment:
This function needs to be updated with the new ABI that this PR contains, I can do this. @seanmc9

Reply:
Committed change [here](https://github.com/JokeDAO/JokeDaoV2Dev/pull/68/commits/375d31f968091a9f8fb9ce67c296bd83bff6b26b), resolving this comment now!